src/container/container.ts

```diff
@@ -27,6 +28,7 @@ class Container implements interfaces.Container {
     private _snapshots: interfaces.ContainerSnapshot[];
     private _metadataReader: interfaces.MetadataReader;
     private _appliedMiddleware: interfaces.Middleware[] = [];
+    private _moduleActivationStore: interfaces.ModuleActivationsStore
```

Comment:
Maybe it should be named `moduleActivationStore`

Reply:
will change